src/web/layouts/StandardLayout.tsx

```diff
@@ -126,7 +130,11 @@ export const StandardLayout = ({ CAPI, NAV }: Props) => {
                         />
                     </LeftColumn>
                     <ArticleContainer>
-                        <StandardHeader CAPI={CAPI} badge={GE2019Badge} />
+                        <StandardHeader
+                            CAPI={CAPI}
+                            badge={GE2019Badge}
```

Comment:
I'm surprised to see `GE2019Badge` hardcoded here? (I know it was in previous PR)

Reply:
I know. There's [a ticket](https://trello.com/c/WpfdQCOq/856-support-badges) to refactor this. It was added directly to this file to get it on the screen quickly but should really live in a lib of some sort.